### PR TITLE
Add blockly scripts.

### PR DESCRIPTION
**Pull request by samelhusseini**
A package to house all dev scripts for Blockly packages. 

Current scripts: 
- build (prod): build dev / prod
```
 * This script:
 *   - Uses webpack to build the src directory in development mode if no
 *   additional arguments are passed.
 *   - Uses webpack to build the src directory in production mode if
 *   ``blockly-scripts build prod`` is called.
```
- start: run dev server
```
 * This script: 
 *   Builds the src and test directories in development mode.
 *   Starts the webpack dev server which watches changes to these directories.
 *   Opens the test page 0.0.0.0:3000/test
 *   Hot reloads the page if any changes are made to the source files.
```
- lint: run eslint on src and test directories
```
 * This script:
 *   Runs eslint on the src and test directories.
```
- clean: remove dist and build directories
```
 * This script:
 *   Deletes the dist and build directories if they exist.
```

Showcasing the use of it in the dev/tools package.

### CODE REVIEW

File: packages/dev/scripts/bin/blockly-scripts.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```
Comment (rachel-fenichel) [inline packages/dev/scripts/bin/blockly-scripts.js:7]: Add fileoverview and author tags

Comment (samelhusseini) [reply packages/dev/scripts/bin/blockly-scripts.js:7]: Done.

**Review by rachel-fenichel — COMMENTED** [review]
All of the code looks reasonable but needs comments and/or fileoverviews.  There's nothing that describes what build, clean, lint, etc will do by default.

**Comment by samelhusseini** [comment]
Sounds good. I've added comments throughout, and file overviews / script overviews.

File: packages/dev/scripts/scripts/start.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const compiler = webpack(config);
+
+// Read the webpack devServer configuration.
+const devServerConfig = config[0].devServer;
+const port = devServerConfig.port;
+const host = devServerConfig.host;
+const page = devServerConfig.openPage;
+const URL = `http://${host}:${port}/${page}`;
+
+// Start the deve server.
```
Comment (rachel-fenichel) [inline packages/dev/scripts/scripts/start.js]: deve -> dev

**Review by rachel-fenichel — APPROVED** [review]
LGTM after spelling fix.

**Pull request merged**